cfgov/legacy/views/complaint.py

```diff
@@ -32,8 +32,7 @@ def get_context_data(self, **kwargs):
         context.update(self.is_ccdb_out_of_date(ccdb_status_json))
 
         context.update({
-            'technical_issues': flag_enabled('CCDB_TECHNICAL_ISSUES'),
-            'ccdb_content_updates': flag_enabled('CCDB_CONTENT_UPDATES')
+            'technical_issues': flag_enabled('CCDB_TECHNICAL_ISSUES')
```

Comment:
Let's leave the comma here so this line doesn't have to change now, or when another flag is added :)

```suggestion
            'technical_issues': flag_enabled('CCDB_TECHNICAL_ISSUES'),
```